### ISSUE TITLE
Testing post-commit resolution.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11040]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1505|
|Type|defect|
|Created|2006-02-25 21:21:59Z|

OH MY GOD THIS BUG IS HORRIBLE

<details><summary>Searchable metadata</summary>

```
trac-id__1505 1505
type__defect defect
reporter__radix radix
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1140902519000000 1140902519000000
changetime__1140902696000000 1140902696000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [31337]) Fixes [#11040](https://github.com/twisted/twisted/issues/11040)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
